src/backend/utils/storage.js

```diff
@@ -45,6 +45,13 @@ module.exports = {
       .exec();
   },
 
+  removeFeedById: async id => {
+    await redis
+      .multi()
+      .srem(feedsKey, id)
```

Comment:
A feed needs to get removed from twice:

1. remove the feed key itself (`t:feed:{feed-id}`)
1. remove the feed's `id` from the feeds set (`t:feeds`)

Reply:
How does this look?
https://github.com/Seneca-CDOT/telescope/blob/9060e955f1acb68c81e23bc0caeef71627db583f/src/backend/utils/storage.js#L48-L56